test/orbit_determination.jl

```diff
@@ -0,0 +1,303 @@
+        # Tracklets
+        @test length(sol.tracklets) == 1
+        @test sol.tracklets[1].radec[1] == radec[1]
+        @test sol.tracklets[end].radec[end] == radec[end]
+        @test issorted(sol.tracklets)
+        # Backward integration
+        @test datetime2days(date(radec[1])) > sol.bwd.t0 + sol.bwd.t[end]
+        @test all( norm.(sol.bwd.x, Inf) .< 2 )
+        @test isempty(sol.t_bwd) && isempty(sol.x_bwd) && isempty(sol.g_bwd)
+        # Forward integration
+        @test datetime2days(date(radec[end])) < sol.fwd.t0 + sol.fwd.t[end]
+        @test all( norm.(sol.fwd.x, Inf) .< 1e9 )
+        @test isempty(sol.t_fwd) && isempty(sol.x_fwd) && isempty(sol.g_fwd)
+        # Vector of residuals
+        @test length(sol.res) == 7
+        @test iszero(count(outlier.(sol.res)))
+        # Least squares fit
+        @test sol.fit.success
+        @test all( sigmas(sol) .< 1e-3 )
+        @test nrms(sol) < 0.13
+        # Scalig factors
+        @test all(sol.scalings .< 1e-5)
+        # Compatibility with JPL
+        JPL = [-0.17932853771087842, 0.8874166708545763, 0.38414497114153867,
+               -0.01755788350351527, -0.005781328974619869, -0.0020073946363600814]
+        @test all(abs.(sol() - JPL) ./ sigmas(sol) .< 0.3)
+
+        # 2008 TC3 entered the Earth's atmosphere around October 7, 2008, 02:46 UTC
+
+        # Fetch optical astrometry
+        radec = fetch_radec_mpc("designation" => "2008 TC3")
+
+        # Parameters
+        params = NEOParameters(coeffstol = Inf, bwdoffset = 0.007, fwdoffset = 0.007)
+
+        # Observations with <1" weight
+        idxs = findall(x -> x < 1, w8sveres17.(radec))
+        # Restricted Orbit Determination
+        sol = orbitdetermination(radec[idxs], params)
+
+        # Values by February 24, 2024
+
+        # Vector of observations
+        @test length(radec) == 883
+        @test numberofdays(radec) < 0.80
+        # Orbit solution
+        @test isa(sol, NEOSolution{Float64, Float64})
+        # Tracklets
+        @test length(sol.tracklets) == 2
+        @test sol.tracklets[1].radec[1] == radec[idxs[1]]
+        @test sol.tracklets[end].radec[end] == radec[idxs[end]]
+        @test issorted(sol.tracklets)
+        # Backward integration
+        @test datetime2days(date(radec[idxs[1]])) > sol.bwd.t0 + sol.bwd.t[end]
+        @test all( norm.(sol.bwd.x, Inf) .< 2 )
+        @test isempty(sol.t_bwd) && isempty(sol.x_bwd) && isempty(sol.g_bwd)
+        # Forward integration
+        @test datetime2days(date(radec[idxs[end]])) < sol.fwd.t0 + sol.fwd.t[end]
+        @test all( norm.(sol.fwd.x, Inf) .< 1e4 )
+        @test isempty(sol.t_fwd) && isempty(sol.x_fwd) && isempty(sol.g_fwd)
+        # Vector of residuals
+        @test length(sol.res) == 20
+        @test iszero(count(outlier.(sol.res)))
+        # Least squares fit
+        @test sol.fit.success
+        @test all( sigmas(sol) .< 2e-5 )
+        @test nrms(sol) < 0.30
+        # Scalig factors
```

Comment:
```suggestion
        # Scaling factors
```
(there's a missing n in `Scaling`)

Reply:
Thank you for noting that. I've corrected all `Scaling` misspellings in that file.